libgnucash/engine/Account.cpp

```diff
@@ -286,6 +286,9 @@ mark_account (Account *acc)
 /********************************************************************\
 \********************************************************************/
 
+static const char*
+is_unset = "unset";
```

Comment:
This causes `ISO C++ forbids converting a string constant to ‘char*’ `

Reply:
Huh, interesting. It has to be `static constexpr const char*`.

Reply:
Huh, interesting. It has to be `static constexpr const char*`. https://godbolt.org/z/ad4vPo7bh

Reply:
Interestingly in the godbolt link you are using `static constexpr char const*` (with `const` after `char` instead of before). I guess this is what the `East const` vs `const West` debate is all about ?

Reply:
https://hackingcpp.com/cpp/design/east_vs_west_const.html

Reply:
Yup. We have so much const west in GnuCash that IMO it's better to stay with it for consistency, though @limitedAtonement used east const in his KVP implementation.